internal/daemon/vendor-specific-plugins/marvell/main.go

```diff
@@ -397,40 +398,25 @@ func getInterfaceName(deviceID string) (string, error) {
 
 // enableIPV6LinkLocal function to enable the IPv6 Link Local Address on the given Interface Name
 // It will return the error
-func enableIPV6LinkLocal(interfaceName string) error {
-	baseNsenterCmd := "nsenter -t 1 -m -u -n -i -- "
-	nmcliCmdStr := baseNsenterCmd + "nmcli con add type ethernet ifname " + interfaceName + " ipv6.method link-local"
-	nmcliCmd := exec.Command("/bin/sh", "-c", nmcliCmdStr)
-	if err := nmcliCmd.Run(); err != nil {
-		return err
-	}
+func enableIPV6LinkLocal(interfaceName string, ipv6Addr string) error {
+	// Ensure to set addrgenmode and toggle link state (which can result in creating
+	// the IPv6 link local address. Ignore errors here.
+	exec.Command("ip", "link", "set", interfaceName, "addrgenmode", "eui64").Run()
+	exec.Command("ip", "link", "set", interfaceName, "down").Run()
```

Comment:
There is no better way then toggling the link to get an IPv6 link local address?